tests/v3_api/common.py

```diff
@@ -615,3 +630,81 @@ def get_admin_client_and_cluster():
     assert len(clusters) > 0
     cluster = clusters[0]
     return client, cluster
+
+
```

Comment:
@soumyalj We should not need a different method of validating ingress for gke cluster . We should be able to use one of the existing methods validate_ingress or validate_ingress_using_endpoint. If there is any waits needed , we can modify these existing validate methods to accommodate for this.